wazo_confd/plugins/line/schema.py

```diff
@@ -58,10 +46,81 @@ class LineSchema(BaseSchema):
         dump_only=True,
     )
 
+    @validates_schema
+    def _validate_only_one_endpoint(self, data, **kwargs):
+        endpoint_sip = data.get('endpoint_sip')
+        endpoint_sccp = data.get('endpoint_sccp')
+        endpoint_custom = data.get('endpoint_custom')
+
+        configured_endpoints = [
+            endpoint
+            for endpoint in [endpoint_sip, endpoint_sccp, endpoint_custom]
+            if endpoint
+        ]
+        if len(configured_endpoints) > 1:
+            raise ValidationError('Only one endpoint should be configured on a line')
+
+    @validates_schema
+    def _validate_multiple_caller_id(self, data, **kwargs):
+        caller_id_name = data.get('caller_id_name')
+        if not caller_id_name:
+            return data
+
+        endpoint_sip = data.get('endpoint_sip')
+        if endpoint_sip:
+            for key, value in endpoint_sip.get('endpoint_section_options'):
+                if key == 'callerid':
+                    raise ValidationError(
+                        'Ambiguous caller ID: line.caller_id_name = {} endpoint_sip.endpoint_section_options["callerid"] = {}',
+                        caller_id_name,
+                        value,
+                    )
```

Comment:
It's not a logger, the string won't be formatted
```suggestion
                    raise ValidationError(
                        f'Ambiguous caller ID: line.caller_id_name = {caller_id_name} endpoint_sip.endpoint_section_options["callerid"] = {value}'
                    )
```